stable/concourse/templates/_helpers.tpl

```diff
@@ -27,9 +27,9 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 
 {{/*
 Create a default fully qualified postgresql name.
-We truncate at 24 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "concourse.postgresql.fullname" -}}
 {{- $name := default "postgresql" .Values.postgresql.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 24 | trimSuffix "-" -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
```

Comment:
If the fullname changes this might be a breaking change to existing users ... so we might want to bump this more than just a patch release ?

Reply:
You're right. Fixing it now.

Reply:
Fixed. Thanks.